generate_images.js

```diff
@@ -0,0 +1,90 @@
+var Canvas = require('canvas')
+  , Image = Canvas.Image;
+var fs = require("fs");
+var path = require('path')
+
+console.log("hello");
```

Comment:
I just removed some of it. I still it find interesting to have the current progress of the operation though so I kept the 'save x/2819 XX.png'